src/Frontend/Modules/MediaLibrary/Twig/Extensions/FrontendHelperExtensions.php

```diff
@@ -3,7 +3,7 @@
 namespace Frontend\Modules\MediaLibrary\Twig\Extensions;
 
 use Frontend\Modules\MediaLibrary\Helper\FrontendHelper;
-use Twig_Markup;
+use Twig\TwigFunction;
 
 class FrontendHelperExtensions extends \Twig_Extension
```

Comment:
Can this \Twig_Extension be imported so we can change this by Twig_Extension

Reply:
Isn't Twig_Extension deprecated? In favour of AbstractExtension? https://github.com/twigphp/Twig/blob/2.x/lib/Twig/Extension.php#L7

Reply:
since this is part of https://github.com/forkcms/forkcms/pull/2918 I'll fix it there. Just also needed that code for this pr

Reply:
fixed in db95773ce